decredUnique/decredUnique_vo.md

```diff
@@ -1,35 +1,41 @@
+New Bitcoin blocks are found and broadcast by proof-of-work miners, who also receive 100% of the Bitcoin block reward, giving miners all the power.
 
-Decred improves on this by utilizing a hybrid proof-of-work - proof-of-stake system. Proof-of-work miners recieve 60% of the Decred block reward for finding blocks, and Proof-of-stake Voters recieve 30% for keeping miners in check.
+Decred utilizes a hybrid proof-of-work - proof-of-stake protocol where 90% of Decred's block reward is split between proof-of-stake Miners and participating stakeholders called Proof-of-stake Voters. Decred Miners receive 60% of the total block reward for finding blocks while Voters receive 30% for keeping the miners in check.
 
 It works like this:
 
-Every 5 minutes a new Decred block is found by a proof-of-work miner. Contained in each block are 5 votes cast by proof-of-stake voters. These votes are used to check the miner behaviour of the previous block.
+Every 5 minutes on average, a new Decred block is found by a proof-of-work miner. Contained in each block are 5 votes cast by 5 proof-of-stake voters. These votes, called block votes, are used to decide whether to accept or reject the previous block.
 
-If a majority deems the pending block unwanted, the miner is stripped of their portion of the block reward as punishment for breaking the rules.
+If a majority decides to reject the block, its miner is stripped of their portion of the block reward as punishment for breaking the rules.
 
-By allowing the rejection of unwanted blocks Decred's hybrid system serves to stop chain-splits and miner attacks.
+By allowing the rejection of blocks, Decred's hybrid protocol provides a mechanism for stakeholders to prevent miner attacks and unwanted hard forks, giving them sovereignty over the Decred chain.
+
+Since Bitcoin employs a pure Proof-of-work protocol, sovereignty over its chain is firmly in the hands of Bitcoin miners, leaving Bitcoin stakeholders powerless.
 
 The differences don't stop there.
 
-10% of the Decred block reward is placed in the network fund. The network fund makes Decred a self-funded open source project with no need for outside capital or an ICO.
+The remaining 10% of the Decred block reward is placed in the Decred treasury. The Treasury makes Decred a self-funded open source project with no need for outside capital or an ICO.
+
+How the funds in the Treasury are spent is also determined by participating stakeholders. Through Politeia, the Decred proposal system, anyone can shape the future of Decred by proposing new ideas for stakeholder consideration. All proposals must be approved by stakeholders, giving them sovereignty over Decred's development.
+
+Bitcoin has zero self-funding and is reliant on outside funding and contributions from those willing to work for free.

```

Comment:
Instead of mentioning Bitcoin again, maybe you can say "without self-funding, a blockchain project is reliant..."